doc/changelog/10-standard-library/09379-splitAt.rst

```diff
@@ -0,0 +1,4 @@
+- Added ``uncons`` and ``splitAt`` functions over vectors,
```

Comment:
Actually, you say here that the function `uncons` is new, but looking at the changes, it seems it existed before and you are just adding lemmas about it...

Reply:
It was added in #8365 but not mentioned in the changelog accordingly. Is it worth putting it here or somewhere?

Reply:
You could either:

1. Add it to the 8.10 changelog (in a separate PR to be backported).
2. Mention it here, but be more accurate, e.g. "Added `splitAt` functions and lemmas about `splitAt` and `uncons` (#9379, follow-up of #8365 which added `uncons` in 8.10+beta1, ...)"